src/TinyContainerBuilder.php

```diff
@@ -124,11 +134,23 @@ public function getInternalContainer(): \Symfony\Component\DependencyInjection\C
 
     public function makePublic(string $service): ContainerBuilderInterface
     {
+        if ($this->makeAllServicesPublic) {
+            throw new \LogicException('Container builder has already been instructed to make all services public.');
+        }
```

Comment:
@harshpatel991 I want to make sure this won't interfere with the usecase you have in mind given how the code already operates in the wild.

Reply:
Yeah this should be fine